### PR TITLE
Enabled active code and tweaked the active code test code.

### PR DESCRIPTION
**Pull request by westyUMass**
Also squashed a few warnings.

### CODE REVIEW

File: test/edu/umass/cs/gnsclient/client/singletests/ActiveCodeTest.java [hunk 1]
```diff
@@ -42,6 +44,15 @@ public ActiveCodeTest() {
       }
     }
   }
+  
+  /**
+   *
+   */
+  @Test
+  public void test_000_ActiveCodeCreateGuids() {
```
Comment (akarthik10) [inline test/edu/umass/cs/gnsclient/client/singletests/ActiveCodeTest.java]: Minor: I would name this method slightly differently as it really isn't doing what it says. Given that DISABLE_ACTIVE_CODE is temporary, this is OK.

File: src/edu/umass/cs/aws/support/Route53.java [hunk 2]
```diff
@@ -62,13 +62,13 @@ public static void main(String[] args) throws Exception {
...
   }
 
-  private void test() {
-    route53.createHostedZone(new CreateHostedZoneRequest()
-            .withName("myDomainName.com")
-            .withCallerReference(CALLER_REFERENCE)
-            .withHostedZoneConfig(new HostedZoneConfig()
-            .withComment("my first Route 53 hosted zone!")));
-  }
+//  private void test() {
```
Comment (akarthik10) [inline src/edu/umass/cs/aws/support/Route53.java]: Any reason for keeping this here? Will this go away later?
(I'm looking at point 5 of coding standards)

Comment (westyUMass) [inline src/edu/umass/cs/aws/support/Route53.java]: The whole file is full of unused code, but that one was generating a warning so I commented it out. Next time I'll probably just suppress the warning.

File: test/edu/umass/cs/gnsclient/client/singletests/ActiveCodeTest.java (hunk 1 above)
Comment (westyUMass) [inline test/edu/umass/cs/gnsclient/client/singletests/ActiveCodeTest.java]: Changed the name.

File: src/edu/umass/cs/aws/support/Route53.java (hunk 2 above)
Comment (v-arun) [inline src/edu/umass/cs/aws/support/Route53.java]: Please don't suppress warnings. Remove the unused code instead.

**Pull request merged**